### PR TITLE
Feature: google analytics

### PR DESCRIPTION
**Pull request by slesaad**
## What am I changing and why

Remove Google Analytics from `.env` since we're no longer using it.
We use Google Tag Manager instead.
Updated repository secrets with the following GTM codes:
```
GOOGLE_TAG_MANAGER_ID=
GOOGLE_TAG_AUTH=
GOOGLE_TAG_PREVIEW=
```

TODO:
- [x] create tag manager container (delegated)
- [x] updated github environment secrets

### CODE REVIEW

**Review by danielfdsilva — APPROVED** [review]

**Review by hanbyul-here — APPROVED** [review]
I can't check environment variables on this repo, but I trust you! 
+ but this pr itself doesn't seem to be necessary. see my comment.

File: .env [hunk 1]
```diff
@@ -23,4 +23,4 @@ MAPBOX_STYLE_URL='mapbox://styles/covid-nasa/ckb01h6f10bn81iqg98ne0i2y'
 GOOGLE_FORM = 'https://docs.google.com/forms/d/e/1FAIpQLSeZrDYP6W3fVOJBwncsLINTdhYsUl9XyoTjNvl7PLBQLV5A0Q/viewform'
 
 # Google analytics tracking code
-# GOOGLE_ANALYTICS_ID=''
+GOOGLE_ANALYTICS_ID='G-738TR1H3KE'
```
[the remark this answers is not in the document]
Comment (danielfdsilva) [reply .env:26]: You're totally right @hanbyul-here . I completely missed this.
We did set up Google Tag Manager, but never configured it.

@slesaad The needed env vars actually are:
```
GOOGLE_TAG_MANAGER_ID=''
GOOGLE_TAG_AUTH=''
GOOGLE_TAG_PREVIEW=''
```

**Review by danielfdsilva — CHANGES_REQUESTED** [review]
Se comment above

File: .env (hunk 1 above)
[the remark this answers is not in the document]
Comment (hanbyul-here) [reply .env:26]: We moved this to env.local (https://github.com/NASA-IMPACT/veda-config/blob/develop/.env.local-sample) We just need to clean up google analytics id.

[the remark this answers is not in the document]
Comment (slesaad) [reply .env:26]: I did update the Google Tag Manager env vars, they're in the repository secrets. I saw the env var and also read that GTM and GA are to be used together, so thought we needed both.

So it's safe to close this PR? maybe I could remove the entry from the .env file itself.

[the remark this answers is not in the document]
Comment (danielfdsilva) [reply .env:26]: that's a great idea @slesaad

**Pull request merged**